cmd/data-node/commands/unsafe_reset_all.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 Gobalsky Labs Limited
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at https://www.mariadb.com/bsl11.
+//
+// Change Date: 18 months from the later of the date of the first publicly
+// available Distribution of this version of the repository, and 25 June 2022.
+//
+// On the date above, in accordance with the Business Source License, use
+// of this software will be governed by version 3 or later of the GNU General
+// Public License.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"code.vegaprotocol.io/vega/datanode/config"
+	"code.vegaprotocol.io/vega/datanode/sqlstore"
+	"code.vegaprotocol.io/vega/logging"
+	"code.vegaprotocol.io/vega/paths"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type UnsafeResetAllCmd struct {
+	config.VegaHomeFlag
+	*config.Config
+}
+
+//nolint:unparam
+func (cmd *UnsafeResetAllCmd) Execute(_ []string) error {
+	log := logging.NewLoggerFromConfig(
+		logging.NewDefaultConfig(),
+	)
+	defer log.AtExit()
+
+	vegaPaths := paths.New(cmd.VegaHome)
+
+	cfgLoader, err := config.InitialiseLoader(vegaPaths)
+	if err != nil {
+		return fmt.Errorf("couldn't initialise configuration loader: %w", err)
+	}
+
+	cmd.Config, err = cfgLoader.Get()
+	if err != nil {
+		return fmt.Errorf("failed to load configuration: %w", err)
+	}
+
+	if err := sqlstore.WipeDatabaseAndMigrateSchemaToLatestVersion(log, cmd.Config.SQLStore.ConnectionConfig, sqlstore.EmbedMigrations,
```

Comment:
I don't think we want the `AndMigrateSchemaToLatestVersion` bit? If I have a fresh postgres instance and I run unsafe_reset_all I would expect it to leave the it alone and not add tables and things. 

For comparison the `unsafe_reset_all` on the blockexplorer just calls `DropAllTablesAndViews()`.

Reply:
good point, I now just drop the DB and recreate an empty DB using some existing code

Reply:
It would be nice if we could just either delete all tables + views, or maybe just the schema (e.g. 'public') then this command wouldn't need DB admin rights but we can add that later